base/all/R/version.R

```diff
@@ -0,0 +1,3 @@
+# Set at package install time, used by pecan.all::pecan_version()
```

Comment:
Conceptually it would be preferable to set this at _build_ time, so that someone could download a prebuilt package binary and still have it provide this information, but the getenv is not evaluated until the package is byte-compiled (which happens during installation). 

(Scripts in `data/` _are_ evaluated at build rather than install, so I considered treating this information as an exported "dataset" rather than an internal variable. I decided against it because that would mean the hash was exported and hence visible to confuse users, but I'd be open to revisiting the idea if other folks think it'd be useful)